tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/lib/wit.js

```diff
@@ -13,6 +13,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
+var witHtmlLocation = require('file-loader!./wit_jupyter.html');
```

Comment:
I'd double check to see if you need the `file-loader!` part. Given your webpack config i wouldn't think that you would need it.

Reply:
the jupyter labextension install fails if i leave it out:

ERROR in ./node_modules/wit-widget/lib/wit_jupyter.html 1:0
Module parse failed: Unexpected token (1:0)

Reply:
Agreed with @tafsiri—perhaps you need to add `module: {rules: rules}` to
the [notebook extension config][1], as in the other two configs?

[1]: https://github.com/tensorflow/tensorboard/blob/ce76dca1432eb06db8544fec169e5a85b81ab995/tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/webpack.config.js#L28-L42

Reply:
i tried that as well - no luck :(

Reply:
Hmm, okay; strange, but I’ll let it be.